docs/UserGuide.md

```diff
@@ -213,17 +213,20 @@ Examples:
 **Note**:
 - Images associated with the client are not deleted and remains in the OS file system.
 
-### Favourite a client: `favourite`
+### Favourite/Unfavourite a client: `favourite`
```

Comment:
You might have to update the table of contents as well

Reply:
Thanks for the reminder!